### ISSUE TITLE
problem installing metran 

### ISSUE THREAD

**Issue by msh855** [pastas/metran#16]
Hi 

I am getting this error when trying to load the package 

`AttributeError: module 'pastas' has no attribute 'stats'`

**Comment by raoulcollenteur** [comment]
have you installed Pastas?

**Comment by msh855** [comment]
yes

**Comment by msh855** [comment]
No, luck. Problem still persists. I think might be dependency issue with matplotlib

**Comment by msh855** [comment]
pastas 0.22 version
